Gemfile.lock

```diff
@@ -0,0 +1,19 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
+    power_assert (1.1.0)
```

Comment:
What's power_assert? I don't require that.

Reply:
It appears that "test-unit >= 3.0" needs it: https://github.com/k-tsj/power_assert and https://rubygems.org/gems/test-unit